src/HttpHeader.h

```diff
@@ -168,6 +169,8 @@ class HttpHeader
     int len;            /**< length when packed, not counting terminating null-byte */
 
 protected:
+    static const size_t UnrestrictedSize = std::numeric_limits<size_t>::max();
```

Comment:
To the extend this is possible:

```suggestion
    static constexpr auto UnrestrictedSize = std::numeric_limits<size_t>::max();
```

This change should actually happen in the adjusted maxLen() implementation. See the change request for that method.

Reply:
This is not needed since I removed maxLen().